### PR TITLE
Remove deprecated code from before v1.0

### PR DESCRIPTION
**Pull request by StorytellerCZ**
* Remove deprecated code from before Meteor 1.0 that  was documented via `XXX COMPAT` in comments.
* Move deprecated packages into the deprecated folder (`deps`, `livedata`, `srp`).

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
Strange, the CircleCI test seem to be getting an outdated version of packages and are failing on code that I have fixed in the latest commit. Is there some cache that needs to be refreshed @filipenevola ?

File: tools/fs/files.ts [hunk 1]
```diff
@@ -215,16 +215,6 @@ export function getToolsVersion() {
...
-
-    // XXX COMPAT WITH 0.9.3
-    const unipackageJsonPath = pathJoin(
-      getCurrentToolsDir(),
-      '..',  // get out of tool, back to package
-      'unipackage.json'
-    );
-    parsed = JSON.parse(readFile(unipackageJsonPath));
-    return parsed.name + '@' + parsed.version;
-
```
Comment (radekmie) [inline tools/fs/files.ts:227]: It should get replaced with an error, right? I mean, _if_ this branch is still reachable, this function will return `undefined`.

**Comment by filipenevola** [comment]
@StorytellerCZ did you reproduce the tests run locally?

I usually run like this:

`./meteor self-test "<failing test name>" --retries 0`

**Comment by filipenevola** [comment]
@StorytellerCZ any news here? Did you try to run the tests locally?

**Comment by StorytellerCZ** [comment]
Sorry, was sick. Will give it a shot this weekend.

File: tools/fs/files.ts (hunk 1 above)
Comment (StorytellerCZ) [reply tools/fs/files.ts:227]: Adding

Comment (StorytellerCZ) [reply tools/fs/files.ts]: Decided to change the attack vector. Momentarily returning the functionality.

File: tools/tests/apps/modules/tests.js [hunk 2]
```diff
@@ -588,6 +588,7 @@ describe("symlinking node_modules", () => {
 
   it("should not break custom import extensions", () => {
     import { jsx } from "jsx-import-test";
+    console.log(jsx);
```
Comment (lgandecki) [inline tools/tests/apps/modules/tests.js:591]:
```suggestion
```

**Pull request closed without merging**